tests/system/python/packages/test_filters.py

```diff
@@ -0,0 +1,710 @@
+        res = conn.getresponse()
+        assert 200 == res.status, "ERROR! Failed to delete filter"
+
+        call_fogbench(wait_time)
+
+        ping_response = verify_ping(foglamp_url)
+        assert 70 == ping_response["dataRead"]
+
+        asset_response = verify_in_asset(foglamp_url, ASSET_NAME_PY35)
+        assert 70 == asset_response["count"]
+
+    def test_filter_python35_by_enabling_disabling_south(self, foglamp_url, wait_time):
+        data = {"schedule_name": "{}".format(self.HTTP_SOUTH_SVC_NAME)}
+        put_url = "/foglamp/schedule/disable"
+        put_request(foglamp_url, put_url, data)
+
+        time.sleep(wait_time)
+
+        get_url = "/foglamp/south"
+        result = get_request(foglamp_url, get_url)
+        assert self.HTTP_SOUTH_SVC_NAME == result["services"][0]["name"]
+        assert "shutdown" == result["services"][0]["status"]
+
+        data = {"name": "py35", "plugin": "python35", "filter_config": {"config": {}, "enable": "true"}}
+        post_request(foglamp_url, "/foglamp/filter", data)
+
+        data = {"pipeline": ["py35"]}
+        put_url = "/foglamp/filter/{}/pipeline?allow_duplicates=true&append_filter=true" \
+            .format(self.HTTP_SOUTH_SVC_NAME)
+        put_request(foglamp_url, urllib.parse.quote(put_url, safe='?,=,&,/'), data)
+
+        url = foglamp_url + urllib.parse.quote('/foglamp/category/{}_py35/script/upload'
+                                               .format(self.HTTP_SOUTH_SVC_NAME))
+        script_path = 'script=@{}/readings35.py'.format(SCRIPTS_DIR_ROOT)
+        upload_script = "curl -sX POST '{}' -F '{}'".format(url, script_path)
+        exit_code = os.system(upload_script)
+        assert 0 == exit_code
+
+        data = {"schedule_name": self.HTTP_SOUTH_SVC_NAME}
+        put_url = "/foglamp/schedule/enable"
+        put_request(foglamp_url, put_url, data)
+
+        time.sleep(wait_time)
+
+        call_fogbench(wait_time)
+
+        ping_response = verify_ping(foglamp_url)
+        assert 80 == ping_response["dataRead"]
+
+        asset_response = verify_in_asset(foglamp_url, ASSET_NAME_PY35)
+        assert 80 == asset_response["count"]
+
+    def test_delete_south_service(self, foglamp_url):
+        conn = http.client.HTTPConnection(foglamp_url)
+        conn.request("DELETE", urllib.parse.quote('/foglamp/service/{}'
+                                                  .format(self.HTTP_SOUTH_SVC_NAME), safe='?,=,&,/'))
+        res = conn.getresponse()
+        assert 200 == res.status, "ERROR! Failed to delete service"
+
+        get_url = "/foglamp/south"
+        result = get_request(foglamp_url, get_url)
+        assert len(result["services"]) == 0
+
+    def teardown_class(cls):
+        try:
+            subprocess.run(["cd {}/tests/system/python/scripts/package && ./reset"
+                           .format(PROJECT_ROOT)], shell=True, check=True)
+            subprocess.run(["cd $FOGLAMP_ROOT/data && rm -rf tests"], shell=True, check=True)
```

Comment:
How about `readings35.py.bak` removal?

Reply:
It is getting replaced with original one i.e. readings35.py, so we do not need to remove it .